server/views/index.ejs

```diff
@@ -0,0 +1,53 @@
+<doctype html>
+<html class="no-js">
+  <head>
+    <meta charset="utf-8">
+    <title>Pathlete</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.css" />
+    <link rel="stylesheet" href="dist/css/app.css">
+  </head>
+  <body ng-app="pathleteApp">
+    <!--[if lt IE 7]>
+      <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+    <![endif]-->
+
+    <div class="container">
+      <div class="header" ng-controller="ToolbarCtrl" ng-show="showing">
+        <ul class="nav nav-pills pull-right">
+          <li><a ng-href="/progress">Progress</a></li>
+          <li><a ng-href="/achievements">Achievements</a></li>
+          <li><a ng-href="/logout" target="_self">Logout</a></li>
+        </ul>
+        <h3 class="text-muted">pathlete</h3>
+      </div>
+      <div ng-view></div>
```

Comment:
Yes, but the ui.router hasn't been merged in yet as far as im aware.  I was going to change it once we actually had ui.router integrated